verbose_debugging_reports.md

```diff
@@ -32,6 +32,9 @@ A source is rejected due to the [destinations per source and reporting site rate
 #### `source-unknown-error`
 System error.
 
+#### `source-output-space-limit`
+A source is rejected due to invalid randomized response configuration.
```

Comment:
Let's avoid mentioning `randomized response configuration` here and instead mention (and link to) both limits in the spec (channel capacity and trigger-state cardinality).